images/cassandra-reaper/tests/smoke-test.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+function cleanup() {
+  helm uninstall k8ssandra-operator -n k8ssandra-operator
+  helm uninstall cert-manager -n cert-manager
+}
+
+trap cleanup EXIT
+
+# list all the storageclasses
+
+kubectl get storageclass
+
+# we have to install cert-manager first
+
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager \
+     --namespace cert-manager --create-namespace --set installCRDs=true
```

Comment:
Hard-coding this means we won't be able to run two of these tests side-by-side. Can you make this namespace `cert-manager-${RANDOM}` to help deconflict?

Reply:
I added RANDOM to every release name and namespace